### PR TITLE
check for error before looking at response

### PR DESCRIPTION
**Pull request by schnie**
Occasionally an error will occur on the requests so we need to check for an error as well as the 200 in the response.  If we don't check for the error you can get a "cannot read statusCode of undefined" error and the app crashes.

### CODE REVIEW

File: index_mlb.js [hunk 1]
```diff
@@ -77,9 +77,10 @@ function getTeamsHierarchy(callback, event) {
 
 
 function createRequest(url, callback) {
+    console.log(url);
```
Comment (rgerard) [inline index_mlb.js]: Could you remove this?

File: .gitignore [hunk 2]
```diff
@@ -36,3 +36,5 @@ Thumbs.db
 
 node_modules/
 .idea/
+*.sublime-workspace
+*.sublime-project
```
Comment (rgerard) [inline .gitignore]: Looks like you need a newline here

Comment (schnie) [inline .gitignore]: sorry about that - fixed!

**Comment by rgerard** [comment]
NPM package has been published as well

**Pull request merged**